### PR TITLE
Simplify audioplayer

### PR DESCRIPTION
**Pull request by mmahalwy**
To do:

- [ ] on click of word with segments
- [x] navigating from home to surah
- [ ] navigating from surah to other surah or to same surah
- [x] changing reciter setting

Goals:
Audioplayer logic is all over the place, within the Track, Audioplayer, and audioplayer module for redux. I wanted to separate the concerns and learn more heavily on redux, making segments and track _dumb_ components. Also, due to the mix of everything, it was hard to track bugs but this should help us with more centralized code.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,6 +1,6 @@
 NODE_ENV=development
 PORT=8000
-API_URL=http://quran.com:3000
+API_URL=http://localhost:3000
```
Comment (mmahalwy) [inline .env]: ill change it back!

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32796

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32797

**Comment by mmahalwy** [comment]
woot

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32798

**Pull request merged**